ONSdigital/dp-frontend-renderer
# Feature/featured content component

### What

Removed the hardcoded sections of the In Focus partial and extracted a dynamic component from it. This will ensure that In Focus content is loaded correctly once the publishing journey is enabled

### How to review

- Check that the code changes make sense
- Ensure the logic for the image works; if there is no image present, it should default to `featured-content-default.svg` as saved in the CDN

- You should be able to run this by running the homepage controller on the branch, `feature/featured-content`. Ensure that you have the flag, `NEW_HOMEPAGE_ENABLED=true` set in `dp-frontend-router` as well. You would need to add featured content to your homepage's `data.json` too. An example entry could be:

```
    "featuredContent": [
      { "title": "Title One", "description": "Test description for Featured Content: Title One", "uri": "/test1", "image": "test1" },
      { "title": "Title Two", "description": "Test description for Featured Content: Title Two", "uri": "/test2", "image": "" }
    ],
```
### Who can review

Anyone but me
